src/lib/api-client.ts

```diff
@@ -42,7 +42,9 @@ class APIClient {
       headers: {
         "Content-Type": "application/json",
         Accept: "application/json",
-        "X-UA": `@topsort/sdk ${version}`,
+        "X-UA": config.userAgent
+          ? `@topsort/sdk ${version} ${config.userAgent}`
```

Comment:
I think it should be fully overridable:
```suggestion
          ? `${config.userAgent}`
```

Reply:
agreed!